.github/workflows/test.yaml

```diff
@@ -16,3 +20,31 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run ${{ matrix.command }}
         run: make ${{ matrix.command }}
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ${{ env.COMPONENT_NAME }}
```

Comment:
Is this even still required?
I guess you can also remove the `cd ${{ env.COMPONENT_NAME }}` calls without this?

Reply:
yes, it's required for now. The git repo repo/dir is called `component-espejo`, but commodore will not be able to compile as it expects `espejo` as the parent dir.